web/src/lib/components/assets/thumbnail/thumbnail.svelte

```diff
@@ -128,7 +131,7 @@
       </div>
 
       <div
-        class="absolute h-full w-full select-none bg-gray-100 transition-transform dark:bg-immich-dark-gray"
+        class="absolute h-full w-full select-none bg-transparent transition-transform dark:bg-immich-dark-gray"
```

Comment:
Why do we need to change this?

Reply:
also related to the change from a `div` with `display: block` to an `a` with `display: flex`

<img width="238" alt="Screenshot 2024-06-07 at 4 23 34 PM" src="https://github.com/immich-app/immich/assets/31512688/aa4f7e67-de76-4eb2-b23c-50dcc6c4257a">

Reply:
previously this wasn't visible, so it didn't matter what the bg colour value was

Reply:
I haven't fully tested it but if we don't need this one then we also don't need `dark:bg-immich-dark-gray` either.

Reply:
nice catch, I tested it and leaving `dark:bg-immich-dark-gray` results in this:

<img width="251" alt="Screenshot 2024-06-07 at 6 18 29 PM" src="https://github.com/immich-app/immich/assets/31512688/82d29eef-7286-446c-bb16-2133fed656b9">

just pushed a commit removing this